modules/pipeline/providers/edgepipeline_register/provider.go

```diff
@@ -36,7 +38,7 @@ type Config struct {
 	ClusterDialEndpoint        string        `file:"cluster_dialer_endpoint" desc:"cluster dialer endpoint"`
 	ClusterAccessKey           string        `file:"cluster_access_key" desc:"cluster access key, if specified will doesn't start watcher"`
 	RetryConnectDialerInterval time.Duration `file:"retry_cluster_hook_interval" default:"1s"`
-	AccessToken                string        `file:"token" env:"EDGE_PIPELINE_ACCESS_TOKEN"`
+	AccessClusterEtcdKeyPrefix string        `file:"cluster_access_key_etcd_prefix" env:"EDGE_PIPELINE_CLUSTER_ACCESS_KEY_ETCD_PREFIX"`
```

Comment:
you should check prefix should not be empty when provider init.

Reply:
also check ending slash

Reply:
checked